pkg/scheduler/apis/config/v1beta3/defaults.go

```diff
@@ -101,6 +102,7 @@ func setDefaults_KubeSchedulerProfile(prof *v1beta3.KubeSchedulerProfile) {
 
 // SetDefaults_KubeSchedulerConfiguration sets additional defaults
 func SetDefaults_KubeSchedulerConfiguration(obj *v1beta3.KubeSchedulerConfiguration) {
+	logger := klog.TODO() // called by generated code which doesn't pass a logger
```

Comment:
It's still a gap, though: if this code gets called in a unit test, it will log via the global klog logger and the log messages will not be visible in the output of a failed unit test.

I agree that this is probably not a big problem.